package.json

```diff
@@ -8,7 +8,7 @@
     "watch": "parcel src/index.html --open --no-autoinstall --port 8080",
     "prebuild": "npm run clean",
     "build": "parcel build src/index.html",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist .cache",
```

Comment:
W jakim to jest celu?

Reply:
Zmiana w pliku package.json była spowodowana tym, że nie były widoczne wprowadzone w kodzie html zmiany po uruchomieniu npm run watch. Stąd ta zmiana.